src/docs/sphinx/docker_info.rst

```diff
@@ -16,27 +16,19 @@ The following instructions apply to the creation of a new compiler image.
     #. Ensure that an Axom image exists on Dockerhub for the desired compiler.
```

Comment:
We should have a comment that most of the time you just need to use the Github Actions build the new docker images.